app/_hub/kong-inc/proxy-cache-advanced/_index.md

```diff
@@ -86,6 +86,13 @@ params:
       datatype: boolean
       description: |
         When enabled, respect the Cache-Control behaviors defined in [RFC7234](https://tools.ietf.org/html/rfc7234#section-5.2).
+    - name: ignore_uri_case
+      required: false
+      default: false
+      value_in_examples: null
+      datatype: boolean
+      description: |
+        If set to `true`, requests will be cached while ignoring case sensitivity in the URI.
```

Comment:
```suggestion
        Determines whether to treat URIs as case sensitive. By default, case sensitivity is enabled. If set to `true`, requests are cached while ignoring case sensitivity in the URI.
```
How about something like this? The initial description feels like it's missing some context.